wgpu/gui/jupyter.py

```diff
@@ -48,6 +48,8 @@ def handle_event(self, event):
             self._pixel_ratio = event["pixel_ratio"]
             self._logical_size = event["width"], event["height"]
 
+        # No need to rate-limit the pointer_move and wheel events;
+        # they're already rate limited by jupyter_rfb in the client.
```

Comment:
Really? To the same rate?

Reply:
No, not the same. Probably less, and may depend on the connection too.